src/protocols/secure_channel/CASESession.cpp

```diff
@@ -119,7 +119,12 @@ using HKDF_sha_crypto = HKDF_sha;
 // Wait at most 30 seconds for the response from the peer.
 // This timeout value assumes the underlying transport is reliable.
 // The session establishment fails if the response is not received within timeout window.
-static constexpr ExchangeContext::Timeout kSigma_Response_Timeout = System::Clock::Seconds16(30);
+static constexpr ExchangeContext::Timeout kInitialExpectedProcessingTime = System::Clock::Seconds16(5);
+
+// Wait at most 30 seconds for the response from the peer.
+// This timeout value assumes the underlying transport is reliable.
+// The session establishment fails if the response is not received within timeout window.
```

Comment:
See comment suggestions above?  I think we could just have that comment, followed by these two values.